applications/directory/src/components/ProfileReview.js

```diff
@@ -91,40 +92,74 @@ const ProfileReview = ({ onSubmit }) => {
-          <Box sx={{ width: '75%' }}>
-            <Typography variant="h3" component="h3">
+          <Box sx={{ marginBottom: '0.875rem' }}>
+            <Typography
+              variant="h2"
+              sx={{
+                // these h2 styles are the same for 4 more headers (2 in this component and 2 in 2 other components)
+                fontSize: '1.25rem',
+                fontFamily: 'gira-sans, sans-serif',
+                fontWeight: 500,
+                marginBottom: '0.875rem',
+                textAlign: 'center'
+              }}
+            >
               Choose what do you want to do with the artist profile:
             </Typography>
-            <RadioGroup
-              row
-              aria-label="Approval status"
-              name="row-radio-buttons-group"
-              value={values.approvalStatus}
+
+            <Box
+              sx={{
+                marginBottom: '0.875rem' // this margin is all over the place on the review page (at least 5 different places)
+              }}
             >
-              <FormControlLabel
-                value="approved"
-                control={<Radio />}
-                label={<span style={{ fontSize: '1rem' }}>{'Approve!'}</span>}
-                name="approvalStatus"
-                onChange={handleChange}
-              />
-              <FormControlLabel
-                value="rejected"
-                control={<Radio />}
-                label={<span style={{ fontSize: '1rem' }}>{'Reject!'}</span>}
-                onChange={handleChange}
-                name="approvalStatus"
-              />
-            </RadioGroup>
+              <RadioGroup
+                row
+                aria-label="Approval status"
+                name="row-radio-buttons-group"
+                value={values.approvalStatus}
+                sx={{
+                  display: 'flex',
+                  width: '80%',
+                  margin: '0 auto',
+                  '& label': {
+                    flex: 1
+                  }
+                }}
+              >
+                <FormControlLabel
+                  value="approved"
+                  control={<Radio />}
+                  label={<span style={{ fontSize: '1rem' }}>{'Approve!'}</span>}
+                  name="approvalStatus"
+                  onChange={handleChange}
+                />
+                <FormControlLabel
+                  value="rejected"
+                  control={<Radio />}
+                  label={<span style={{ fontSize: '1rem' }}>{'Reject!'}</span>}
+                  onChange={handleChange}
+                  name="approvalStatus"
+                />
+              </RadioGroup>
+            </Box>
+            {/* <Divider light /> */}
           </Box>
 
           {values.approvalStatus === 'rejected' && (
             <Fade in={values.approvalStatus === 'rejected'}>
               <Box
                 sx={{
                   '& p': {
```

Comment:
Need to relocate this `& p` styling to the appropriate element (if there is one?).

Reply:
these are styles for error messages which are not hard coded